.npmignore

```diff
@@ -1,4 +1,3 @@
-modules
```

Comment:
otherwise we can't use a github fork in package.json, as `dist/index.js` isn't available in the git repository and we need _something_...